charts/newrelic-infra-operator/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+description: A Helm chart to deploy the New Relic Infrastructure Kubernetes Operator.
+name: newrelic-infra-operator
+version: 0.1.0
+appVersion: 1.4.0
```

Comment:
This still needs to be fixed.

Reply:
I'll repoint to 0.1.0-pre since we have that released (although outdated)